### PR TITLE
Remove un-necessary dependencies

### PR DESCRIPTION
**Pull request by andreaTP**
Follow up from:
https://typesafe.slack.com/archives/CQUASEUDV/p1625068565235800

### CODE REVIEW

File: samples/valueentity-counter/package.json [hunk 1]
```diff
@@ -7,11 +7,7 @@
     "npm": ">=6.0.0"
   },
   "dependencies": {
-    "@grpc/grpc-js": "^1.3.0",
```
Comment (andreaTP) [inline samples/valueentity-counter/package.json:10]: Note for the reviewers, this project is built and tested in CI, so we are confident that the dependencies are not needed.

File: npm-js/create-akkasls-entity/template/base/package.json [hunk 2]
```diff
@@ -7,11 +7,7 @@
     "npm": ">=6.0.0"
   },
   "dependencies": {
-    "@grpc/grpc-js": "^1.3.0",
-    "@grpc/proto-loader": "^0.1.0",
-    "@lightbend/akkaserverless-javascript-sdk": "0.7.0-beta.11",
-    "google-protobuf": "^3.0.0",
-    "protobufjs": "^6.11.2"
+    "@lightbend/akkaserverless-javascript-sdk": "0.7.0-beta.13"
```
Comment (pvlugter) [inline npm-js/create-akkasls-entity/template/base/package.json:10]: Side note: we probably want this version to be updated automatically at some point...

**Review by pvlugter — APPROVED** [review]
LGTM

I think some of the samples had explicit dependencies from when they depended on the SDK directly as a path, which is probably where this originated.

File: npm-js/create-akkasls-entity/template/base/package.json (hunk 2 above)
Comment (andreaTP) [reply npm-js/create-akkasls-entity/template/base/package.json:10]: yes, that work is not finalized yet. Going to work on it.

**Pull request merged**